agent/internal/client/interface.go

```diff
@@ -59,6 +59,10 @@ type SharedCommunicator interface {
 	GetDistroView(context.Context, TaskData) (*apimodels.DistroView, error)
 	// GetDistroAMI gets the AMI for the given distro/region
 	GetDistroAMI(context.Context, string, string, TaskData) (string, error)
+	// GetDistroByName returns a distro given a distro ID.
+	GetDistroByName(context.Context, string) (*restmodel.APIDistro, error)
```

Comment:
Does this function get it by name or id (the comment above says id but the name says name)

EDIT: I see that the method was already implemented and it gets it by id rather than name like the name suggests. Could you rename it to say GetDistroByDistroID maybe? Or GetDistroByID (if it makes sense that the ID in question is the Distro's)?

Reply:
yeah that makes a lot of sense! thanks for pointing it out